styleguide/css/MODERN-CSS.md

```diff
@@ -0,0 +1,138 @@
+![Barsoom](http://barsoom.se/barsoom.png)
+
+# Modern CSS styleguide
+
+Also see [our top-level styleguide](/styleguide): it explains how and when to follow our styleguides, and how to change them.
+
+This guide is specifically aimed at new projects using modern web tools.
+
+---
+
+### Frameworks and libraries
+
+Avoid frameworks and libraries as much as you can.
+
+Use SVG icons instead of font icons. They both have their pros and cons, but SVG is a little bit more flexible (allows for colors and animations). [Icon Fonts vs SVG – Clash of the Icons](https://www.lambdatest.com/blog/its-2019-lets-end-the-debate-on-icon-fonts-vs-svg-icons/)
+
+
+### Use [prettier](https://prettier.io/)
+
+This will force our CSS to be consistently formatted.
+
+
+### Use [stylelint](https://stylelint.io/)
+
+TODO: We still need to write a stylelint config, and make sure it's compatible with prettier so that it doesn't complain over rules that would be fixed by prettier.
+
+
+### Use [postcss](https://postcss.org/) with [postcss-preset-env](https://preset-env.cssdb.org/)
+
+[PostCSS Preset Env](https://preset-env.cssdb.org/) lets you convert modern CSS into something most browsers can understand, determining the polyfills you need based on your targeted browsers or runtime environments, using [cssdb](https://cssdb.org/).
+
+
+### Use CSS Modules
```

Comment:
I'm not sure about this for reasons given in https://github.com/barsoom/auctionet/pull/147/files#r636957608.

Let's discuss it sometime after the deadline work we're doing now, when I have more mindspace. Possibly we want to wait until Fred joins us to make big decisions about changing how we do CSS?

Reply:
CSS modules are basically the same as BEM, but automated.